pyGHDL/requirements.txt

```diff
@@ -1,2 +1,3 @@
 pydecor>=2.0.1
-pyVHDLModel==0.10.2
+#pyVHDLModel==0.10.2
```

Comment:
Before merging, we should release pyVHDLModel 0.10.3, publish it to PyPI and replace this modification with just bumping 0.10.2 to 0.10.3,

Reply:
It's fixed to `v0.10.3`.